### DISCUSSION TITLE
Help updating Unifi Network controller LXC

### DISCUSSION THREAD

**Discussion by Scoff123** [tteck/Proxmox#3387] [Help Needed] [answered]
I am trying to update my Unfii network controller which was originally created using the script from this repo a few weeks ago, however I don't seem to be able to successfully update to the latest version. I have run apt update && apt upgrade from the console of my controller, and the update pulls down correctly, but the upgrade from 8.2.93 to 8.3.32 does not happen. I have verified that 8.2.93 is still the installed version. I am still quite new to Proxmox and LXC's as well as Unifi setup, so it may be something obvious, but I don't understand why the update is not happening. I have attached the apt log from the update and upgrade command if it helps. 
[apt.txt](https://github.com/user-attachments/files/16254104/apt.txt)

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/3381
